src/main/steps/c100-rebuild/typeofhearing/urgent/template.njk

```diff
@@ -0,0 +1,17 @@
+{% extends "common/template.njk" %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+
+{% block page_content %}
+<h1 class="govuk-heading-xl govuk-!-margin-top-7">{{title}}</h1>
+<p class="govuk-body">{{paragraph1}}</p>
+<p class="govuk-body">{{paragraph2}}</p>
+<p class="govuk-body">{{paragraph3}}</p>
```

Comment:
Please check if we can move paragraph1, paragraph2, paragraph3 inside content.ts fields